### PR TITLE
Fix typo in the README

### PR DESCRIPTION
**Pull request by TriinK**
Small typo fix

### CODE REVIEW

**Comment by AlbertoPdRF** [comment]
Hey @TriinK, thanks for your interest on our project and in addressing #333! :star_struck:

However, it would be better if we discussed first which badges to add, and also use the correct data for them. Feel free to start a discussion about this on the issue.

Besides, good catch on the wrong wording! Maybe we can leave this PR just for that and, after we have a plan on how to proceed regarding the badges, open a new one for that. How does that sound?

**Comment by TriinK** [comment]
Sure, no problem :) I removed the badges and only left that one small typo fix in :)

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
   <strong>Meet people while doing what you enjoy!</strong>
 </div>
 
+<br/>
+
```
Comment (AlbertoPdRF) [inline README.md]:
```suggestion
```
This should be removed for now too

**Review by AlbertoPdRF — APPROVED** [review]
LGTM! Thanks for catching the typo and making your first contribution to Tisn @TriinK! :female_detective:

**Pull request merged**